.github/workflows/release.yaml

```
@@ -122,7 +122,7 @@ jobs:
       - name: Install and run trunk (Web)
         if: ${{ matrix.platform == 'web' }}
         run: |
-          cargo binstall trunk --no-confirm
+          cargo binstall trunk wasm-bindgen-cli wasm-opt --no-confirm
```

```suggestion
          cargo binstall --no-confirm trunk wasm-bindgen-cli wasm-opt
```
Probably nice to move the flag earlier now, but not important.